Ruby-Project/person_class.rb

```diff
@@ -19,6 +20,12 @@ def correct_name
     @name
   end
 
+  def add_rental(date, person)
+    rental = Rental.new(date, person)
+    @rental << rental
+    book.add_rental(rental)
+  end
```

Comment:
As mentionned previously, the only thing that's necessary is to create the rental in this method, something like this should do: 
```
def add_rental(book, date)
    Rental.new(date, book, self)
end
```